tests/ably/setup/test-app-setup.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
We could probably remove some extraneous stuff from this, looks like most of it isn't being used

Reply:
Removed!